release-checklist.md

```diff
@@ -21,6 +21,7 @@
 - [ ] `git branch -D master-test`
 - [ ] `git push origin :master-test`
 - [ ] Announce availability in #engineering and on the forum here: https://forum.blockstack.org/c/releases (Include a link to the changelog)
+- [ ] Announce availability to the community via Jenny Mith <jenny@blockstack.com> and Mitchell Cuevas <mitchell@blockstack.com>. Provide a succinct list of changes, the like to the changelog and latest release as well as a desired date for the community update.
```

Comment:
Should it be "the link to the changelog" instead?

Reply:
I think a link is not sufficient enough. They want two things mainly: 1) a date when this should go out to the community and 2) a message that should go out to the community. Just providing a link to the changelog won't be enough. We should clearly community in brief why and what was changed as well as what the impact is. It's something that will be really hard for Jenny or Mitchell to get from the pure technical changelog is my argument.

Reply:
I was only referring to the spelling error: "like" in place of "link". Otherwise looks good to me.

Reply:
Hah, of course, that needs to be fixed :)